utils/src/main/kotlin/Utils.kt

```diff
@@ -123,7 +124,7 @@ fun filterVersionNames(version: String, names: List<String>, project: String? =
 
     return filteredNames.filter {
         // startsWith("") returns "true" for any string, so we get an unfiltered list if "project" is "null".
-        it.startsWith(project.orEmpty())
+        it.startsWith(packageName.orEmpty())
```

Comment:
Please also update the comment directly above to use "packageName" instead of "project".

Does the comment below also need to be updated?